javascript/shared/third_parties/crypto_init.yml

```diff
@@ -0,0 +1,19 @@
+type: shared
+languages:
+  - javascript
+patterns:
+  - import $<!>$<_>$<...> from "crypto"
+  - pattern: |
+      $<METHOD>($<NAME>)
+    filters:
+      - variable: METHOD
+        values:
+          - require
+          - import
```

Comment:
This isn't picking up `require` statements and I'm not sure why 🤔 e.g. 

```
const myCrypto = require("crypto")
myCrypto.createHmac(...)
```

Reply:
^ This was due to duplicate variable names in the shared rules (something we want to handle in the future). For now, we've added prefixes like `<RULE_NAME>_MY_VAR` to all variable names in the shared rules. 
Thanks to @didroe for the debugging on this 🙏